CONTRIBUTING.md

```diff
@@ -35,3 +35,10 @@ Must be one of the following:
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
 - **test**: Adding missing tests or correcting existing tests
+
```

Comment:
Afaik it should be just a single empty line above and below a heading as per markdownlint, let's see what others think 🤔 
```suggestion
```